### PR TITLE
Arg mention

### PR DESCRIPTION
**Pull request by hyanwong**
Starts to address https://github.com/tskit-dev/tskit/issues/1922

### CODE REVIEW

**Comment by hyanwong** [comment]
I was wondering whether to add to `python/README.rst`, but that seems like a very brief description. I don't know what the normal format is for this, and therefore whether we should expand it at all.

File: README.md [hunk 1]
```diff
@@ -10,7 +10,13 @@
 Succinct tree sequences are a highly efficient way of storing a set of related DNA
 sequences by encoding their ancestral history as a set of correlated trees along the
 genome. The tree sequence format is output by a number of software libraries and programs
-(such as [msprime](https://github.com/tskit-dev/msprime), [SLiM](https://github.com/MesserLab/SLiM), [fwdpp](http://molpopgen.github.io/fwdpp/), and [tsinfer](https://tsinfer.readthedocs.io/en/latest/)) that either simulate or infer the evolutionary history of genetic sequences.
+(such as [msprime](https://github.com/tskit-dev/msprime),
+[SLiM](https://github.com/MesserLab/SLiM),
+[fwdpp](http://molpopgen.github.io/fwdpp/), and
+[tsinfer](https://tsinfer.readthedocs.io/en/latest/)) that either simulate or infer
+the evolutionary history of genetic sequences. Technically, the ancestral history that
+can be encapsulated in a tree sequence is sometimes referred to as an "ancestral
```
Comment (jeromekelleher) [inline README.md:18]: "often" is probably more accurate than "sometimes"

File: README.md [hunk 2]
```diff
@@ -10,7 +10,13 @@
...
+recombination graph" or ARG.
```
Comment (jeromekelleher) [inline README.md:19]: How about 
> The evolutionary history of genetic sequences is often technically referred to as an Ancestral Recombination Graph (ARG). Succinct tree sequences are fully compatible with this formulation, and tskit is a powerful platform for processing ARGs.

**Review by jeromekelleher — COMMENTED** [review]
LGTM, some suggested text.

Re the README.rst, it's the front-page on PyPI and intentionally pretty short to avoid duplication. I think the idea was to point to the stable documentation there and the latest in the github README.

**Comment by jeromekelleher** [comment]
Would you mind crossing off #1943 since you're in there also please @hyanwong?

**Comment by hyanwong** [comment]
Fixes #1943

**Review by jeromekelleher — APPROVED** [review]
LGTM

**Pull request merged**